packages/contracts/deploy/015-OVM_L1StandardBridge.deploy.ts

```diff
@@ -27,33 +28,25 @@ const deployFn: DeployFunction = async (hre) => {
     return
   }
 
-  const OVM_L1ETHGateway = await getDeployedContract(hre, 'OVM_L1ETHGateway', {
+  const OVM_L1StandardBridge = await getDeployedContract(hre, 'OVM_L1StandardBridge', {
     signerOrProvider: deployer,
   })
 
+  const l1MessengerAddress = await Lib_AddressManager.getAddress('Proxy__OVM_L1CrossDomainMessenger')
+
   // NOTE: this initialization is *not* technically required (we only need to initialize the proxy)
   // but it feels safer to initialize this anyway. Otherwise someone else could come along and
-  // initialize this.
-  await OVM_L1ETHGateway.initialize(
-    Lib_AddressManager.address,
-    predeploys.OVM_ETH
+  // initialize this. We'll set all the addresses to 0 so that this copy is unusable.
+  await OVM_L1StandardBridge.initialize(
+    constants.AddressZero,
```

Comment:
I like what we are doing here, but actually I think this would technically allow the contract to be re-initialized because we check that the address of the messenger is nonzero as our only-initialize-once check in `OVM_L1StandardBridge`.

Reply:
Good catch!